__tests__/suggestions.test.js

```diff
@@ -221,4 +221,11 @@ describe('Suggestions', function() {
 
     expect($el.find('.bar').length).to.equal(4);
   });
+
+  test('should trigger the click handler on touchStart', function () {
+    const onTagClickedStub = stub();
+    const $el = mount(mockItem({ onTagClickedStub: onTagClickedStub }));
+    const instances = $el.find('span');
+    instances.map(instance => expect(instance.simulate('touchStart')));
```

Comment:
simulate `touchStart`event on one of the span and check if onTagClickedStub was triggered